redash/models/__init__.py

```diff
@@ -64,6 +65,58 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+@python_2_unicode_compatible
+class TableMetadata(TimestampMixin, db.Model):
```

Comment:
Both this and the columns table needs to include `org_id` reference.

Reply:
Added, though I'm not sure if it needs to be used in a specific way or do you mean in case it needs to be referenced in the future?